### PR TITLE
all: update to new cloud.google.com package path

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

**Comment by broady** [comment]
Also fixes an incorrect import path in an appengine sample (it imported cloud/datastore instead of appengine/datastore)

File: docs/appengine/datastore/consistency/consistency.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ package sample
 
 import (
 	"golang.org/x/net/context"
-	"google.golang.org/cloud/datastore"
+	"google.golang.org/appengine/datastore"
```
Comment (rakyll) [inline docs/appengine/datastore/consistency/consistency.go]: Why is this importing appengine now?

Comment (broady) [inline docs/appengine/datastore/consistency/consistency.go]: It was wrong before (see comment above)

Comment (rakyll) [inline docs/appengine/datastore/consistency/consistency.go]: Ack

**Comment by rakyll** [comment]
LGTM

**Comment by broady** [comment]
Waiting for build.

**Pull request merged**